### ISSUE TITLE
Jump-back button for ALT-click quoting

### ISSUE THREAD

**Issue by quintesse** [gitterHQ/gitter#680]
Right now you could be in the middle of a long list of new messages that you're reading through, you click on a reference which moves you to some unknown part of the log and then good luck finding your way back again to where you were.
It would be nice to have a button that would show up after clicking the reference that would take you back to the line you clicked on.

**Comment by mydigitalself** [comment]
It's not quite the same, but there is a big overlay button thing that appears when you jump fairly high up in the conversation that will return you to the bottom. Great suggestion with returning you to where you were though.

**Comment by quintesse** [comment]
@mydigitalself indeed, but like you say it's not the same, but I'm referring exactly to such a similar solution! Maybe the same button can be re-utilized for that purpose.

PS: with respect to that button, you already say "fairly high up", why is that? Why not display it immediately whenever you scrolled up even a moderate amount? I often find myself looking for it after scrolling up and find it a tiny bit of a nuisance that I have to scroll down myself.

**Comment by grahamperrin** [comment]
Interesting. I sometimes multi-quote within a single composition before posting.
